src/utils/dna-holo-hash-map.ts

```
@@ -0,0 +1,438 @@
+   * @param callbackfn - Function to execute for each entry
+   * @param thisArg - Optional value to use as 'this' when executing the callback
+   */
+  forEach(
+    callbackfn: (value: V, key: K, map: Map<K, V>) => void,
+    // This 'any' is inherited from the Map interface.
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    thisArg?: any,
+  ): void {
+    return this._map.forEach((v, k) => {
+      callbackfn(v, this._decodeKey(k) as K, this);
+    }, thisArg);
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)